DataFormats/HGCalReco/src/classes_def.xml

```diff
@@ -3,6 +3,9 @@
   <class name="ticl::Trackster" ClassVersion="3">
    <version ClassVersion="3" checksum="3878166595"/>
   </class>
+  <class name="ticl::Trackster" ClassVersion="4">
```

Comment:
The exact syntax should be (see https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning):

```
  <class name="ticl::Trackster" ClassVersion="4">
   <version ClassVersion="4" checksum="522492601"/>
   <version ClassVersion="3" checksum="3878166595"/>
  </class>
```

Reply:
Thanks for pointing me to the docs, I fixed it in the last commit.